cli/src/main.rs

```diff
@@ -46,12 +48,15 @@ fn main() {
                     println!("{monitor}: {path:?}");
                 }
             }
-            IpcResponse::Ok => (),
+            IpcResponse::Ok => println!("daemon response: OK"),
```

Comment:
I think this message here is too general. Maybe it's better to improve it in another separated PR, what do you think?